pkg/doc/verifiable/presentation.go

```diff
@@ -306,6 +306,27 @@ func NewPresentation(vpData []byte, opts ...PresentationOpt) (*Presentation, err
 		return nil, err
 	}
 
+	return newPresentation(vpRaw, vpOpts)
+}
+
+// NewUnverifiedPresentation decodes Verifiable Presentation from bytes which could be marshalled JSON or
+// serialized JWT. It does not make a proof check though. Can be used for purposes of decoding of VP stored in a wallet.
+// Please use this function with caution.
+func NewUnverifiedPresentation(vpBytes []byte) (*Presentation, error) {
```

Comment:
We should have a TODO to re-do naming (e.g., with Parse prefix).

Reply:
Yeah, I will file the issue soon.